jetbrains: drop bundled plugins (#470147)
NixOS/nixpkgs: doc/release-notes/rl-2605.section.md

- All Log4Shell vulnerability scanners were removed, as they were all unmaintained upstream and are no longer relevant given that the vulnerability has been fixed upstream for several years.

> - Plugins for the Jetbrains IDEs have been removed from nixpkgs. It is no longer possible to use `jetbrains.plugins.addPlugins` with a plugin name/ID to add that plugin to an IDE. However generally addings plugins to IDEs is still supported, using a list of derivations.

Reviewer: 1. [one sentence per line](https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#one-sentence-per-line)
2. maybe this should be two notes?
3. wording suggestions

```suggestion
- Plugins for the JetBrains IDEs have been removed from Nixpkgs.

- `jetbrains.plugins.addPlugins` no longer supports plugin names or ID strings.
  You can still use `addPlugins` with plugin derivations, such as plugins packaged outside of Nixpkgs.
```
